src/libraries/System.IO.Compression/tests/CompressionStreamUnitTests.Deflate.cs

```diff
@@ -219,5 +219,95 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
                 return base.WriteAsync(buffer, offset, count, cancellationToken);
             }
         }
+
+        [Theory]
+        [MemberData(nameof(ZLibOptionsRoundTripTestData))]
+        public async Task RoundTripWithOptions(string testFile, ZLibCompressionOptions options)
```

Comment:
Why does this test need to be duplicated for each of deflate, gzip, and zlib rather than it living in the shared tests?

Reply:
I think I can make it shared

Reply:
Done